app/views/foreman_tasks/tasks/dashboard/_latest_tasks_in_error_warning.html.erb

```diff
@@ -11,7 +11,7 @@
       <td class="ellipsis"><%= link_to task.humanized[:action], defined?(main_app) ? main_app.foreman_tasks_task_path(task.id) : foreman_tasks_task_path(task.id) %></td>
       <td><%= task.state %></td>
       <td><%= task.result %></td>
-      <td><%= _('%s ago') % time_ago_in_words(task.ended_at) %></td>
+      <td><%= task.ended_at ? _('%s ago') % time_ago_in_words(task.ended_at) : _('Paused') %></td>
```

Comment:
That is a good suggestion, I would be 👍 to `started_at` which avoids the paused/completed edge case and still presents useful info.